src/Microsoft.OData.Edm/Schema/Interfaces/IEdmPrimitiveType.cs

```diff
@@ -180,6 +180,11 @@ public enum EdmPrimitiveTypeKind
         /// Represents a TimeOfDay type
         /// </summary>
         TimeOfDay,
+
+        /// <summary>
+        /// Represents a Primitive type
+        /// </summary>
+        PrimitiveType,

```

Comment:
What is the difference between this value and "None"?

Reply:
They are different. 

"None" means we don't know the primitive type.  It's always used in Reader/Parser.

For example, a primitive type named "Edm.Unknown" will return `EdmPrimitiveTypeKind.None`.